packages/@lwc/engine-server/src/__tests__/fixtures/for-each-block/child-nested-complex/modules/x/for-each-nested-complex/for-each-nested-complex.html

```diff
@@ -0,0 +1,11 @@
+<template lwc:render-mode="light">
+    <template for:each={relatedListData} for:item="list" if:true={relatedListData}>
+        <template for:each={list.recordIds} for:item="recordId">
+            <template for:each={list.fields} for:item="field" for:index="index">
+                <x-child key={field}>
+                    <div>{field}</div>
+                </x-child>
+            </template>
+        </template>
+    </template>
+</template>
```

Comment:
Before the change in this PR the output is this:

```js
const __lwcGenerateShadowSlottedContent_xChild_0 = (list, __unused__, recordId, __unused__, field, index) => function __lwcGenerateShadowSlottedContent_xChild_0
```
The double `__unused__` led to a compilation error.

After this change it now appears as:

```js 
const __lwcGenerateShadowSlottedContent_xChild_0 = (list, __unused__, recordId, field, index) => function __lwcGenerateShadowSlottedContent_xChild_0
```